dhis-2/dhis-api/src/main/java/org/hisp/dhis/datavalue/DataValueStore.java

```diff
@@ -109,6 +109,21 @@
     DataValue getDataValue( DataElement dataElement, Period period, OrganisationUnit source,
         CategoryOptionCombo categoryOptionCombo, CategoryOptionCombo attributeOptionCombo );
 
+    /**
+     * Returns a DataValue.
+     *
+     * @param dataElement the DataElement of the DataValue.
+     * @param period the Period of the DataValue.
+     * @param source the Source of the DataValue.
+     * @param categoryOptionCombo the category option combo.
+     * @param attributeOptionCombo the attribute option combo.
+     * @param includeDeleted Include deleted data values
+     * @return the DataValue which corresponds to the given parameters, or null
+     *         if no match.
+     */
+    DataValue getDataValue( DataElement dataElement, Period period, OrganisationUnit source,
```

Comment:
Following what we said during Thursday meeting this is a "find" type of method but I guess there are already exist other and naming these now inconsistent is also bad. We should try to update this at some point to follow the get/find pattern.